packages/backend/src/tokens/__tests__/organizationMatcher.test.ts

```diff
@@ -70,20 +70,20 @@ describe('OrganizationMatcher', () => {
       });
     });
 
-    it('should find personal account', () => {
+    it('should find personal workspace', () => {
       const matcher = new OrganizationMatcher({
-        personalAccountPatterns: ['/account'],
+        personalWorkspacePatterns: ['/account'],
       });
       const url = new URL('http://localhost:3000/account');
       expect(matcher.findTarget(url)).toEqual({
-        type: 'personalAccount',
```

Comment:
Should probably keep the tests for `personalAccount` to make sure that the deprecated value works when bumping the version, avoiding breaking changes.